.github/workflows/create_tag.yml

```diff
@@ -254,21 +268,8 @@ jobs:
 
           def get_artifacts_tf():
             artifacts = []
-            GITHUB_TOKEN = os.environ.get('GITHUB_TOKEN')
-            url = f'https://api.github.com/repos/TessellDevelopment/{REPO}/pulls/{PR_ID}/files?per_page=100'
-            headers = {'Authorization': f'Bearer {GITHUB_TOKEN}'}
-            res = requests.get(url, headers=headers)
-            files_data = res.json()
-          
-            modified_folders = set()
-            for file_data in files_data:
-              file_path = file_data['filename']
-              folder_path = file_path.split('/')[0] 
-              if folder_path == 'convoy.yaml':
-                continue 
-              modified_folders.add(folder_path)
-          
-            print("Modified folders:")
+            modified_folders = get_directories()
+            print("Terraform Folders")
             for folder in modified_folders:
               if '.github' in folder or len(folder) == 0:
```

Comment:
You already skip this in `get_directories` ... so not required anymore
Also rename `get_directories` to `get_tf_modules_folders`